### ISSUE TITLE
How to debug issue with dropping session / exit 1?

### ISSUE THREAD

**Issue by colinjcahill** [Wowu/gotodir#20]
Hi, I have had this problem periodically with gotodir since I installed it:  sometimes when I go to a directory, it kills my terminal session.  I use iTerm 2, so what it looks like is that the tab I am in disappears.  This would happen if there is an `exit 1` in the current process.  

I'm not sure how to start debugging this.  I don't see a log file in the node_module.  any suggestions?

**Comment by wowu** [comment]
Hi, so it crashes when you use `to <directory-name>` command? Is it nondeterministic, or happens for particular directories?

**Comment by colinjcahill** [comment]
> Hi, so it crashes when you use `to <directory-name>` command? Is it nondeterministic, or happens for particular directories?

yes that's right.  so far I haven't found any pattern.  here is what I am seeing:

- Happens with any entry in gotodir list
- Happens independent of where the entry is in the list
- Is repeatable in a given session.  So if I restore a crashed iterm tab and attempt `to <dirname>` again, it will crash again
- I will confirm, but I believe that the issue is session-dependent.  So if I crash going to a folder name in one session, I believe that I can access it properly with the same command in another terminal tab.

I think the best clue so far is that I believe it is session specific.  I'll add more details to this comment as I encounter them
